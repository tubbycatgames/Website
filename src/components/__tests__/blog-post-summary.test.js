import React from "react"
import renderer from "react-test-renderer"

import BlogPostSummary from "../blog-post-summary"

describe("Blog Post", () => {
  it("renders a summary for a blog post", () => {
    const tree = renderer.create(
      <BlogPostSummary creation="2004-09-22" slug="A Page" title="A Post!" />
    )
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
