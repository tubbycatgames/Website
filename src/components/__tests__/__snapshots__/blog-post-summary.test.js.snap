// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Post renders a summary for a blog post 1`] = `
<a
  href="A Page"
>
  <h2
    className="title"
  >
    A Post!
  </h2>
  <small>
    2004, September 22nd
  </small>
</a>
`;
